frame/nft/README.md

```diff
@@ -0,0 +1,37 @@
+# Pallet-NFT
+A pallet containing the most essential functionalities to create, and manage assets with NFT characteristics.
+
+## Overview
+The NFT framework pallet provides developers with fundamental tools including:
+* Minting an asset into a wallet
+* Edit, customize and store an asset's unique attributes
+* Transfer of ownership from one user to another
+* Burn asset from a wallet
+* Inspect the attributes and owner of an asset
+
+## Implementations
+Trait implementations for Pallet-NFT are provided by composable_traits and composable_support
+
+| Name                 | Description                        |
+|----------------------|------------------------------------|
+| FinancialNFTProvider | Handle AccountId                   |
+| NFTClass             | Default ClassId type for NFT's     |
+| Create               | Handle creation of assets          |  
+| Inspect              | Inspect information of an asset    |
+| Mutate               | Change an asset's information      |
+| Transfer             | Facilitate transfer of ownership   |
+
+## Extrinsics
+| Name                      | Caller | Description                                                                                                                                         |
+|---------------------------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| create_class              | Admin  | Create a new class of assets, ensuring the class doesn't already exists                                                                             |
+| owner                     | User   | Display the owner of an asset or vice versa                                                                                                         |
```

Comment:
note sure, owner is not Extrinsic as per substrate term afaik. is more like metadata about data.

Reply:
Noted. 👍